test/math.jl

```diff
@@ -992,9 +992,11 @@ end
         @test isnan_type(T, tanh(T(NaN)))
         for x in Iterators.flatten(pcnfloat.([H_SMALL_X(T), T(1.0), H_MEDIUM_X(T)]))
             @test tanh(x) ≈ tanh(big(x)) rtol=eps(T)
-            @test tanh(-x) ≈ tanh(big(-x)) rtol=eps(T)
+            @test tanh(-x) ≈ -tanh(big(x)) rtol=eps(T)
```

Comment:
wouldn't this potentially be less accurate (assuming that BigFloat tanh is more accurate than inv)?

Reply:
What do you mean? `-tanh(big(x))` and `tanh(big(-x))` should be different by a maximum of `1` bigfloat ULP (and in reality will probably be exactly equal)

Reply:
ah, right, I was reading the hypot PR just before this and confused myself